client/components/Popover/PopoverButton.tsx

```diff
@@ -38,9 +38,9 @@ const PopoverButton = (props: Props) => {
 			<Popover
 				aria-label={ariaLabel}
 				className={className}
-				unstable_portal={true}
 				tabIndex={0}
 				{...popover}
+				modal={true}
```

Comment:
with the change and with `unstable_portal`

![image](https://user-images.githubusercontent.com/34730449/223754446-c0272dad-5395-4c6a-9e83-8b7ccfd2d734.png)